src/app/server/Server.h

```diff
@@ -254,6 +254,16 @@ class Server
      */
     void DispatchShutDownAndStopEventLoop();
 
+    /**
+     * This function generate the SoftwareFault Event when a software fault takes place on the Node.
+     */
+    void DispatchSoftwareFaultEvent(uint32_t eventId);
```

Comment:
What is "eventId" here?

Reply:
My point is that this needs to be documented.  Is this an EventId?  Or an actual uint32_t?  If the latter, how does it get interpreted?

Reply:
moved to cluster server